application/forms/Config/UserGroup/AddMemberForm.php

```diff
@@ -1,129 +1,52 @@
 <?php
-/* Icinga Web 2 | (c) 2015 Icinga Development Team | GPLv2+ */
+/* Icinga Web 2 | (c) 2022 Icinga Development Team | GPLv2+ */
```

Comment:
The year must not change, the file isn't new after all. But you should change `Icinga Development Team` to `Icinga GmbH`.